### ISSUE TITLE
VRF tickets should be a chain

### ISSUE THREAD

**Issue by nicola** [filecoin-project/lotus#2778]
We are still working on this.

The current ticket does not use the ticket from the previous epoch as input.

This means that the ticket chain is not "specific" to a chain anymore. This has implications on the validity of SealRandomness.

**Comment by nicola** [comment]
We should not do this since this would make the ticket for leader election grindable. We can only do this if we have a separate ticket chain (hence a second VRF).

This has implications and we should discuss this in the next dev meeting.

**Comment by nikkolasg** [comment]
Currently computing a ticket takes the previous ticket **only** if there are no new drand entry in the block !
* this [code](https://github.com/filecoin-project/lotus/blob/b8bbbf3ea3b186e658be9a8011fd6827b13aa3e5/miner/miner.go#L432) computes the ticket with a boolean `haveNewEntries`. Only if it is false, it adds the previous ticket to the drawing randomness logic
* That flag is given [here](https://github.com/filecoin-project/lotus/blob/b8bbbf3ea3b186e658be9a8011fd6827b13aa3e5/miner/miner.go#L362) and is true when there are new drand entry to include in the new block.

 Basically, with drand frequency = filecoin frequency, it seems like the previous ticket is never used ?

**Comment by nicola** [comment]
Working on a PR: https://github.com/filecoin-project/lotus/pull/3510

**Comment by arajasek** [comment]
Fixed by #3510
